ui/app/models/node-attributes.js

```diff
@@ -9,8 +9,13 @@ export default Fragment.extend({
   attributes: attr(),
 
   attributesStructured: computed('attributes', function() {
-    // `unflatten` doesn't sort keys before unflattening, so manual preprocessing is necessary.
     const original = this.get('attributes');
+
+    if (!original) {
+      return;
+    }
```

Comment:
What causes `attributes` to be falsy here? I thought the default value for a fragment was an empty object.

Reply:
The attribute of the fragment can still be falsy:

https://github.com/hashicorp/nomad/blob/cb35b874d66aade6e15ee18285bfea581231769f/ui/app/serializers/node-attributes.js#L4-L6